game/models.py

```diff
@@ -172,17 +177,18 @@ def make_standard_deck():
 class Dungeon:
     """Handle deck, room and player creation and interaction"""
 
-    def __init__(self, cards=make_standard_deck(), seed=None):
+    def __init__(self, cards=None, seed=None):
```

Comment:
well, the dungeon progresses through the deck in a certain way based on the PRNG, so to recreate a gamestate, you should feed it the same deck (set of cards), same seed, and then the same (partial) sequence of inputs. But yes, this design would let you do this rapidly (save inputs and replay them "behind the scenes").

The idea behind taking the card set as a variable is to make it easier to test different decks (different values for face cards, different amounts of cards (two decks together), etc)